renku/cli/dataset.py

```diff
@@ -76,6 +104,50 @@
     data/
       my-dataset/
         datafile
+
+Listing all files within a dataset.
+
+.. code-block:: console
+
+    renku dataset ls-files
+
+will yield:
+
+.. code-block:: console
+
+    ADDED                AUTHORS    DATASET        PATH
+    -------------------  ---------  -------------  ---------------------------
+    2019-02-28 16:48:09  sam        my-dataset     ...my-dataset/addme
+    2019-02-28 16:49:02  sam        my-dataset     ...my-dataset/weather/file1
+    2019-02-28 16:49:02  sam        my-dataset     ...my-dataset/weather/file2
+    2019-02-28 16:49:02  sam        my-dataset     ...my-dataset/weather/file3
+
+Sometimes you want to filter the files. For this we use ``--dataset``,
+``--include`` and ``--exclude`` flags:
+
+.. code-block:: console
+
+    renku dataset ls-files --include "file*" --exclude "file3"
+
+which will yield:
+
+.. code-block:: console
+
+    ADDED                AUTHORS    DATASET     PATH
+    -------------------  ---------  ----------  ---------------------------
+    2019-02-28 16:49:02  sam        my-dataset  ...my-dataset/weather/file1
+    2019-02-28 16:49:02  sam        my-dataset  ...my-dataset/weather/file2
+
+
+Removing a file from a dataset:
+
+.. code-block:: console
+
+    $ renku dataset unlink my-dataset --include "weather/*"
```

Comment:
1) It's not needed. 2) It will try to unlink all files within a dataset. 

I added more examples in relation to this question.

Thank you for the review.